### ISSUE TITLE
FIrefox OS Simulator not running

### ISSUE THREAD

**Issue by swarnava** [mozilla/r2d2b2g#151]
I download a fresh version of b2g from https://ftp.mozilla.org/pub/mozilla.org/labs/r2d2b2g/r2d2b2g-windows.xpi and installed it on my Firefox 17. but however if i click on start button its showing that Firefox OS is running but nothing happening actually and the busy icon keep rotating!

**Comment by vladmaniac** [comment]
I can also vouch for this issue. Repro with Build identifier: Mozilla/5.0 (X11; Linux i686; rv:19.0) Gecko/19.0 Firefox/19.0 given the steps from swarnava. Should we restart the browser, it works but it should work from start, unless we are prompted for a restart.

Nothing in the console about this - add-on issue maybe?

**Comment by ghost** [comment]
Same here...

**Comment by juninho** [comment]
Same here:
Mozilla/5.0 (Windows NT 6.1; WOW64; rv:17.0) Gecko/17.0 Firefox/17.0

**Comment by juninho** [comment]
Today, with the same Firefox version of yesterday, it's working...

**Comment by aurium** [comment]
Not working for me. I had installed r2d2b2g today, and had restarted Firefox, but still not working. I'm using Debian Stable (Linux 2.6.32-5-686), and the newest Firefox directly from Mozilla (17.0).

When i click on the toggle runner button nothing happens, and no log is generated to the console.
The "add directory" and "add URL" buttons does nothing. (no log too)

I tried to run `Simulator.addAppByDirectory()` from firebug. Nothing happens and it returned `undefined`.

For the url I tried this: http://scientific-calculator.appspot.com/manifest.webapp ... and nothing happens.

**Comment by mykmelez** [comment]
I see reports here for both Windows and Linux, which are likely to be different problems. The Linux problems in particular, which have been tracked in #105 are likely to have been resolved in the latest build, which I uploaded to the distribution server on Saturday. And the Windows issues may also be resolved.

So try installing the latest build, and let me know if the Simulator starts!

[latest version of Simulator for Linux](https://ftp.mozilla.org/pub/mozilla.org/labs/r2d2b2g/r2d2b2g-linux.xpi)

[latest version of Simulator for Windows](https://ftp.mozilla.org/pub/mozilla.org/labs/r2d2b2g/r2d2b2g-windows.xpi)
